apps/kilocode-docs/docs/providers/aistupidlevel.md

```diff
@@ -0,0 +1,102 @@
+
+Instead of manually choosing between GPT-4, Claude, Gemini, or other models, AIStupidLevel automatically selects the optimal model for your task based on continuous performance monitoring.
+
+## Getting an API Key
+
+1. **Sign Up:** Go to [https://aistupidlevel.info](https://aistupidlevel.info) and create an account
+2. **Navigate to Router:** Click on the "Router" section in the dashboard
+3. **Add Provider Keys:** Add your API keys for the providers you want to use (OpenAI, Anthropic, Google, xAI, etc.)
+4. **Generate Router Key:** Create a router API key that Kilo Code will use
+5. **Copy the Key:** Copy your AIStupidLevel router API key
+
+## Available Routing Strategies
+
+AIStupidLevel offers different "auto" models that optimize for specific use cases:
+
+| Model | Description | Best For |
+|-------|-------------|----------|
+| `auto` | Best overall performance across all metrics | General-purpose tasks |
+| `auto-coding` | Optimized for code generation and quality | Software development, debugging |
+| `auto-reasoning` | Best for complex reasoning and problem-solving | Deep analysis, mathematical problems |
+| `auto-creative` | Optimized for creative writing quality | Content creation, storytelling |
+| `auto-cheapest` | Most cost-effective option | High-volume, budget-conscious tasks |
+| `auto-fastest` | Fastest response time | Real-time applications, quick queries |
+
+## Configuration in Kilo Code
+
+1. **Open Kilo Code Settings:** Click the gear icon (<Codicon name="gear" />) in the Kilo Code panel.
+2. **Select Provider:** Choose "AIStupidLevel" from the "API Provider" dropdown.
+3. **Enter API Key:** Paste your AIStupidLevel router API key into the "AIStupidLevel API Key" field.
+4. **Select Model:** Choose your desired routing strategy from the "Model" dropdown (e.g., `auto-coding`, `auto-reasoning`, etc.).
+
+## How It Works
+
+When you make a request through Kilo Code:
+
+1. **AIStupidLevel analyzes** current model performance from continuous benchmarks
+2. **Selects the optimal model** based on your chosen routing strategy
+3. **Routes your request** using your configured provider API keys
+4. **Returns the response** with metadata about which model was selected
+
+The router automatically:
+- Avoids models experiencing performance degradation
+- Routes to cheaper models when performance is comparable
+- Provides transparent routing decisions in response headers
+
+## Key Features
+
+- **Degradation Protection:** Automatically avoids models with performance issues
+- **Cost Optimization:** Routes to cheaper models when performance is comparable
+- **Provider Diversity:** Access models from OpenAI, Anthropic, Google, xAI, DeepSeek, and more through one API
+- **Transparent Routing:** Response headers show which model was selected and why
+- **Performance Tracking:** Dashboard shows your usage, cost savings, and routing decisions
+- **Enterprise SLA:** 99.9% uptime guarantee with multi-region deployment
```

Comment:
Rather than including the marketing copy here, could you include a link to your own pages?